test/upload-test-images.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+
+: ${DOCKER_REPO_OVERRIDE:?"You must set 'DOCKER_REPO_OVERRIDE', see DEVELOPMENT.md"}
+
+export KO_DOCKER_REPO=${DOCKER_REPO_OVERRIDE}
+IMAGE_PATHS_FILE="$(find $(dirname $0)/image_paths.txt)"
```

Comment:
Using `find` is more robust in the way that the script can be run from both eventing/ and eventing/test folder. Not a big deal but I think it's useful. Using just relative path means I can only run it from eventing/test folder.

Reply:
If you're using `dirname $0` the script will also be runnable from eventing/ and from eventing/test, since its location will be relative to this script.

Reply:
Right.